app/script/conversation/ConversationMapper.js

```diff
@@ -22,21 +22,53 @@
 'use strict';
 
 /**
- * @typedef {object} SelfStatusUpdate
- * @property {number=} archived_timestamp
- * @property {number=} cleared_timestamp
- * @property {number=} ephemeral_timer
- * @property {number=} message_timer
- * @property {number=} last_event_timestamp
- * @property {number=} last_read_timestamp
- * @property {number=} last_server_timestamp
- * @property {boolean=} otr_archived
+ * @typedef {object} ConversationData
+ * @property {string[]=} access
+ * @property {string=} access_role
+ * @property {string=} creator
+ * @property {string=} id
+ * @property {string=} last_event
+ * @property {string=} last_event_time
+ * @property {ConversationMembers=} members
+ * @property {?number=} message_timer
+ * @property {string=} name
+ * @property {?string=} team
+ * @property {number} type
+ */
+
+/**
+ * @typedef {object} ConversationMembers
+ * @property {OtherMember[]} others
+ * @property {Member} self
+ */
+
+/**
+ * @typedef {object} Member
+ * @property {string=} hidden_ref
+ * @property {boolean=} hidden
+ * @property {string=} id
  * @property {string=} otr_archived_ref
- * @property {boolean=} otr_muted
+ * @property {boolean=} otr_archived
  * @property {string=} otr_muted_ref
- * @property {boolean=} muted_state
- * @property {number=} status
- * @property {number=} verification_state
+ * @property {boolean=} otr_muted
+ * @property {ServiceRef=} service
+ */
+
+/**
+ * @typedef {object} OtherMember
+ * @property {string} id
+ * @property {number} status
+ */
+
+/**
+ * @typedef {object} SelfStatusUpdate
```

Comment:
This one seems to be incomplete, I believe we can pass a few more properties to be updated in the self status

Reply:
Good catch! Fixed.